sdks/python/apache_beam/io/external/xlang_jdbcio_it_test.py

```diff
@@ -144,11 +144,11 @@ def test_xlang_jdbc_read(self):
       result = (
           p
           | 'Read from jdbc' >> ReadFromJdbc(
+              table_name=table_name,
```

Comment:
Ideally we would keep both sets of tests, one that relies on just the table name, and one that sets the SELECT/INSERT statement. We can leave this for a follow-up though if you file a jira and add a TODO

Reply:
Done.